spec/05-classes-and-objects.md

```diff
@@ -1104,7 +1104,7 @@ lazily.  The `new $m$\$cls` constructor is evaluated
 not at the point of the object definition, but is instead evaluated
 the first time $m$ is dereferenced during execution of the program
 (which might be never at all). An attempt to dereference $m$ again in
-the course of evaluation of the constructor leads to a infinite loop
+the course of evaluation of the constructor leads to an infinite loop
 or run-time error.
```

Comment:
This is just a typo PR, but the language of the sentence is unnecessarily dense, probably because it tries to say something general. "When in the course of human events,..."  I would say: `again during evaluation of the constructor may lead`, which is sufficiently wishy-washy.